Source/Core/Core/ConfigManager.h

```diff
@@ -14,6 +14,9 @@
 #include "Common/Common.h"
 #include "Common/CommonTypes.h"
 
+// X.h defines Always to be 2, which causes problems with enum HideCursor
+#undef Always
```

Comment:
Undefining something defined by a system library just to avoid a name conflict seems very risky and is a potential source of bugs. Why not rename `Always` from `enum ShowCursor` to something else instead?

Reply:
Someone had suggested I mimic what another file has done to undefine X.h's `Always`, but that scenario had quite a number of enums so it was probably a more reasonable instance to use it. For this enum it does make sense to change it and I'll do so.